modelling/src/main/java/org/axonframework/modelling/command/AbstractRepository.java

```diff
@@ -462,6 +470,18 @@ public Builder<T> subtype(@Nonnull Class<? extends T> subtype) {
             return this;
         }
 
+        /**
+         * Sets the {@link SpanFactory} implementation to use for providing tracing capabilities.
```

Comment:
Note that the `builder` methods throughout also define the default values. Hence, I'd expect all the infra components where you have added this default field to have a change in the `builder`'s JavaDoc too.